.github/workflows/publish-auto.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Auto
+
+on: [push]
+  branches-ignore:
+    # No canary deploys for branches opened by dependabot
+    - 'dependabot/**'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
Do we use this right now? If not, let' remove it.

Reply:
This is necessary because auto create commits which contain `ci skip)` when it updates the changelog and bumps the version in the `stable` branch. This guard is present to prevent auto from triggering an infinite loop (creating a commit, rerunning auto, etc)

![Image 2021-07-20 at 9 14 22 PM](https://user-images.githubusercontent.com/9020979/126415047-4901a7b6-d4ad-46f0-9650-c6effd4b4eb2.jpg)

https://intuit.github.io/auto/docs/build-platforms/github-actions